boards/shields/st_b_lcd40_dsi1_mb1166/boards/stm32f769i_disco.conf

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2024 Linumiz
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_MEMC=y
+CONFIG_STM32_LTDC_RGB888=y
+CONFIG_HEAP_MEM_POOL_SIZE=65536
+# Initialize after LTDC and MIPI-DSI
+CONFIG_DISPLAY_OTM8009A_INIT_PRIORITY=87
```

Comment:
This is specific to shield revisions that use the OTM8009A controller, so it should be in a dedicated `stm32f769i_disco.conf` in `boards/shields/st_b_lcd40_dsi1_mb1166/boards/st_b_lcd40_dsi1_mb1166`